hack/build.sh

```diff
@@ -63,7 +63,7 @@ build-docker() {
         -e GOARCH="${GOARCH}"                                             \
         -e CGO_ENABLED=0                                                  \
         -e GO111MODULE=on                                                 \
-        -e GOPROXY=https://goproxy.io                                     \
+        -e GOPROXY="${GOPROXY}"                                           \
```

Comment:
Could you please update the `Makefile` and `env.sh` for environmental variable `GOPROXY `? THX

Reply:
This env has already exists before. We set it to `https://goproxy.io`